src/Symfony/Component/Form/Extension/Validator/Constraints/FormValidator.php

```diff
@@ -26,10 +26,10 @@ class FormValidator extends ConstraintValidator
     /**
      * {@inheritdoc}
      */
-    public function validate($form, Constraint $constraint)
+    public function validate($form, Constraint $formConstraint)
     {
-        if (!$constraint instanceof Form) {
```

Comment:
The `$constraint` name is also used below when the constraints configured through the `constraints` option are iterated, but we do not want to overwrite the reference to the constraint that is validated by this constraints.